client/common/client.c

```diff
@@ -424,20 +393,22 @@ static BOOL client_cli_authenticate_raw(freerdp* instance, rdp_auth_reason reaso
 {
 	static const size_t password_size = 512;
 	const char* auth[] = { "Username: ", "Domain:   ", "Password: " };
-	const char* authPin[] = { "Username: ", "Domain:   ", "Pin:      " };
+	const char* authPin[] = { "Username: ", "Domain:   ", "Smartcard-Pin:      " };
```

Comment:
The spaces in the messages were arranged so that all prompts are vertically aligned, could you adjust that ?